api/src/cctp_calls.rs

```diff
@@ -106,6 +107,65 @@ pub fn compute_pks_threshold_hash(
     Ok(pks_threshold_hash)
 }
 
+pub fn compute_msg_to_sign_prehashed_bt(
```

Comment:
If we refactor everything to use the WithdrawalCertificate class/data structure then we can rename this simply compute_msg_to_sign and delete the other one